warehouse/query-core/src/main/java/datawave/core/iterators/IteratorThreadPoolManager.java

```diff
@@ -70,15 +131,23 @@ private ThreadPoolExecutor createExecutorService(int maxThreads, String name) {
         return pool;
     }
 
-    private int getMaxThreads(final String prop, AccumuloConfiguration conf) {
-        if (conf != null) {
+    private int getIntPropertyValue(final String prop, int defaultValue, AccumuloConfiguration config) {
+        if (config != null) {
             Map<String,String> properties = new TreeMap<>();
-            conf.getProperties(properties, k -> Objects.equals(k, prop));
+            config.getProperties(properties, k -> Objects.equals(k, prop));
```

Comment:
confused between this method and the getLongPropertyValue.  Why not use config.get(prop) ?

Reply:
The note on AccumuloConfiguration.get() says that it is inefficient for keys that are not properties and recommends this approach.